Sources/SKCore/CompilationDatabase.swift

```diff
@@ -185,12 +211,22 @@ extension JSONCompilationDatabase: Codable {
 }
 
 extension JSONCompilationDatabase {
-  public init(directory: AbsolutePath, _ fileSystem: FileSystem = localFileSystem) throws {
+  /// Loads the compilation database located in `directory`, if any.
+  /// returns: `nil` if `compile_commands.json` was not found
```

Comment:
Let’s use docc markup here. Same for all the other `return:` lines

```suggestion
  ///
  /// - Returns: `nil` if `compile_commands.json` was not found
```